extensions/file-manager/package.json

```diff
@@ -54,6 +55,14 @@
       "required": true,
       "type": "checkbox"
     },
+    {
+      "default": false,
+      "description": "Show the file hidden by the chflags hidden command.",
```

Comment:
I would just say "hidden files", no need to elaborate that it is via chflags, as I believe there's only one to show/hide files on macos, aside from dotfiles